lineman/app/components/mixins.scss

```diff
@@ -95,8 +95,8 @@ $block-border-color: darken($block-color, 10%);
 }
 
 @mixin boxTiny {
-  width: 20px;
-  height: 20px;
+  width: 40px;
```

Comment:
Sorry but this is not allowed to be changed. Tiny is 20px that's the rules. We ahve 20, 30 and 50 px things.. Can you use one of those sizes?